fe/fe-core/src/main/antlr4/org/apache/doris/nereids/DorisParser.g4

```diff
@@ -395,6 +395,7 @@ unsupportedLoadStatement
     | RESUME ALL ROUTINE LOAD                                                       #resumeAllRoutineLoad
     | STOP ROUTINE LOAD FOR label=multipartIdentifier                               #stopRoutineLoad
     | SHOW ALL? ROUTINE LOAD ((FOR label=multipartIdentifier) | wildWhere?)         #showRoutineLoad
+    | SHOW ROUTINE LOAD IN ALL                                                      #showRoutineLoad
```

Comment:
why not just `show routine load` to get all jobs.
we can known whether it is an "all" context if we check  the "current db":
* if a db is selected by `use xxx`, means show jobs for the db
* no db is selected, means we need to show all

Reply:
the previous impl. we can eliminate this limit to get the work done
```
mysql> SHOW ROUTINE LOAD;
ERROR 1046 (3D000): errCode = 2, detailMessage = No database selected
```